### PR TITLE
places, concepts: GND closeMatch corrections

### PR DESCRIPTION
**Pull request by rerowep**

### CODE REVIEW

File: rero_mef/marctojson/do_gnd_places.py [hunk 1]
```diff
@@ -270,29 +270,42 @@ def trans_gnd_match(self):
...
-                                match.setdefault("identifiedBy", []).append(
+                                identified_by.insert(
+                                    0,
                                     {
                                         "type": "uri",
                                         "value": subfield_0,
-                                    }
+                                    },
                                 )
+                                if other_source:
```
Comment (PascalRepond) [inline rero_mef/marctojson/do_gnd_places.py:288]: This is always False?

File: rero_mef/marctojson/do_gnd_concepts.py [hunk 2]
```diff
@@ -253,29 +247,42 @@ def trans_gnd_match(self):
...
-                                match.setdefault("identifiedBy", []).append(
+                                identified_by.insert(
+                                    0,
                                     {
                                         "type": "uri",
                                         "value": subfield_0,
-                                    }
+                                    },
                                 )
+                                if other_source:
```
Comment (PascalRepond) [inline rero_mef/marctojson/do_gnd_concepts.py:265]: Same as below.

**Review by PascalRepond — APPROVED** [review]

File: rero_mef/marctojson/do_gnd_places.py (hunk 1 above)
Comment (rerowep) [reply rero_mef/marctojson/do_gnd_places.py:288]: It is set in line 295.

File: rero_mef/marctojson/do_gnd_concepts.py (hunk 2 above)
Comment (rerowep) [reply rero_mef/marctojson/do_gnd_concepts.py:265]: It is set in line 272.

**Review by Garfield-fr — APPROVED** [review]

**Pull request merged**